### PR TITLE
fix: Fix path for java sdk client after it was moved around

### PR DESCRIPTION
**Pull request by pratik151192**
### Description

We changed the directory structure and renamed a few directories on our java client sdk to incorporate more variants of examples and not piling them under the same directory. Reference: https://github.com/momentohq/client-sdk-java/pull/297

The build is expected to fail until we merge the aforementioned PR. We can restart the build once it's merged and it will succeed.

### CODE REVIEW

**Review by cprice404 — APPROVED** [review]
you can mark this PR as "draft" until it's ready to be merged.

LGTM once the other PR goes in and this build goes green.

**Comment by pratik151192** [comment]
Re-ran builds which succeeded and merged

**Pull request merged**
